packages/devtools/lib/src/inspector/inspector.dart

```diff
@@ -163,6 +186,40 @@ class InspectorScreen extends Screen {
     final InspectorTreeWeb detailsInspectorTree =
         inspectorController.details.inspectorTree;
 
+    serviceManager.hasRegisteredService(
```

Comment:
it would be good to put the business logic about how we figure out whether expand and collapse are supported in the inspector_controller instead of the Screen class.